microblog/spec/features/blog_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe 'Blogs features' do
+    let(:view_blogs) do
+        visit('/blogs')
+    end
+
+    let(:create_blog) do
+        Blog.create!(title: 'kiran', body: 'good', status: 1)
+        view_blogs
+    end
+
+    describe 'viewing the index' do
+        it 'lists all of the blogs' do
```

Comment:
why you have not used context here?

Reply:
both describe and context has same use and context is removed as top level function according to rspec docs
https://github.com/rspec/rspec-core/blob/395866dc0df64b5719429f803e2a056275bc7bd3/features/Upgrade.md#context-is-no-longer-a-top-level-method